### ISSUE TITLE
ChakraCore.dll should be available directly for use

### ISSUE THREAD

**Issue by mubbasher** [Microsoft/ChakraCore#1281]
I'm on Windows 7 and VS2012 and tried to build the ChakraCore but no luck.

I prefer to have just ChakraCore.dll , plug that into my C# app and start using it ...

Can you guys provide the X86 and X64 ChakraCore.dll files...

Please don't make stupid like Linux that we have to build it all the time from source.

**Comment by dilijev** [comment]
/cc @liminzhu 

See #85 

https://www.myget.org/feed/chakracore-preview/package/nuget/Microsoft.ChakraCore

We are in the process of adding documentation about this to the README and Wiki as well. See PR #1253

**Comment by dilijev** [comment]
Duplicate of #1180

**Comment by dilijev** [comment]
@mubbasher By the way, I see you're using VS2012 on Windows 7. We do support Windows 7 but we do not support building with VS 2012 (see https://github.com/Microsoft/ChakraCore/wiki/Building-ChakraCore).

It should work, but we haven't tested whether binaries built with VS 2013 will run on systems that do not have VS 2013 installed. Please let us know if you have any problems.

**Comment by obastemur** [comment]
Installing https://www.microsoft.com/en-us/download/details.aspx?id=48145 should take care of the C++ runtime dependencies.

**Comment by mubbasher** [comment]
@dilijev Thanks...  @obastemur Thanks.
I loved this and will be witching our app from V8 to ChakraCore. This is a huge relief ... 
I was able to run the C# HelloWorld app using "msvcp140.dll" and "vcruntime140.dll" for X64 machine.
I don't know if installing the runtime will have impact on VS2012 or previous app. For now I will keep it like that.
I used the Build setting as , X64, Release mode for C# app and it works fine :+1:
